environments/production/applications/locals.tf

```diff
@@ -10,6 +10,6 @@ locals {
     "fpo-search",
     "frontend",
     "signon",
-    "terraform-1.5.5-python3",
```

Comment:
I think this was put here by accident.

We're not using this lock when pushing new docker images for the terraform app